Paper/main.tex

```diff
@@ -127,15 +127,14 @@ \section{RESNET explanation}
 
 \subsection{Training without transfer learning}
 
-The idea behind training the network from scratch is that the network would be more tailored to the given task since it has no previous information stored in the network. This contrasts with transfer learning since there prior information is present.
+The idea behind training the network from scratch is that the network would be more tailored to the given task since it has no previous information stored in the network. This contrasts with transfer learning since there prior information is present. As a model the ResNet18 architecture was used. This architecture was chosen since it is a good compromise between network size and performance.
```

Comment:
this should address #84

Reply:
And what about the 10 class point?

Reply:
I think the point about 10 classes was more about the paragraph *Dataset details*. since there the sentence was like *This leads to 10 classes* i rewrote that part.